pxr/usd/usdLux/lightAPI.h

```diff
@@ -359,10 +420,120 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
+    /// 
+    /// - MeshLightAPI
+    /// - DiskLight
+    /// - RectLight
+    /// - SphereLight
+    /// - CylinderLight
+    /// - (deprecated) GeometryLight
+    /// 
+    /// <center><b>
+    /// sizeFactor<sub>area</sub> = worldSpaceSurfaceArea(light)
+    /// </b></center>
+    /// 
+    /// ### DistantLight:
+    /// 
+    /// For distant lights, we first define 𝛳<sub>max</sub> as:
+    /// 
+    /// <center><b>
+    /// 𝛳<sub>max</sub> = clamp(toRadians(distantLightAngle) / 2, 0, 𝜋)
+    /// </b></center>
+    /// 
+    /// Then we use the following formula:
+    /// 
+    /// * <i>if 𝛳<sub>max</sub> = 0:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> = 1
+    /// </b></center>
+    /// 
+    /// * <i>if 0 < 𝛳<sub>max</sub> ≤ 𝜋 / 2:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> = sin²𝛳<sub>max</sub> ⋅ 𝜋
+    /// </b></center>
+    /// 
+    /// * <i>if 𝜋 / 2 < 𝛳<sub>max</sub> ≤ 𝜋:</i>
+    /// <center><b>
+    /// sizeFactor<sub>distant</sub> =
+    /// (2 - sin²𝛳<sub>max</sub>) ⋅ 𝜋
+    /// </b></center>
+    /// 
+    /// This formula is used because it satisfies the following two properties:
+    /// 
+    /// 1. When normalize is enabled, the received illuminance from this light
+    /// on a surface normal to the light's primary direction is held constant
+    /// when angle changes, and the "intensity" property becomes a measure of
+    /// the illuminance, expressed in lux, for a light with 0 exposure.
+    /// 2. If we assume that our distant light is an approximation for a "very
+    /// far" sphere light (like the sun), then (for
+    /// *0 < 𝛳<sub>max</sub> ≤ 𝜋/2*) this definition agrees with the
+    /// definition used for area lights - ie, the total power of this distant
+    /// sphere light is constant when the "size" (ie, angle) changes, and our
+    /// sizeFactor is proportional to the total surface area of this sphere.
+    /// 
+    /// ### Other Lights
+    /// 
+    /// The above taxonomy describes behavior for all built-in light types.
+    /// (Note that the above is based on schema *family* - ie, `DomeLight_1`
+    /// follows the rules for a `DomeLight`, and ignores `normalize`.)
+    /// 
+    /// Lights from other third-party plugins / schemas must document their
+    /// expected behavior with regards to normalize.  However, some general
+    /// guidelines are:
+    /// 
+    /// - Lights that either inherit from or are strongly associated with one of
+    /// the built-in types should follow the behavior of the built-in type
+    /// they inherit/resemble; ie, a renderer-specific "MyRendererRectLight"
+    /// should have it's size factor be it's world-space surface area
+    /// - Lights that are boundable and have a calcuable surface area should
+    /// generally follow the rules for an Area Light, and have their
+    /// sizeFactor be their world-space surface area
+    /// - Lights that are non-boundable and/or have no way to "intuitively"
```

Comment:
typo + suggestion

> Lights that are non-boundable or otherwise have no meaningful notion of a size should ignore this attribute.

Logic: if they do not ignore size, then by definition size is meaningful.

Reply:
Oops, fixed all these: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4R445

Reply:
I assume the typo was the "calcuable" you also noted [here](https://github.com/PixarAnimationStudios/OpenUSD/pull/3182#discussion_r1702859881)?

As for the logic note - I agree!  ...but I'm not clear on what change you wanted to make as a result.  Did you mean that we should just eliminate the sentence because it's obviously true? Or just eliminate the use of the word "meaningful"?

Suggested change: https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-bdeeccf2a8491e83067bc3c1e20e5a696612da9b31d1e3b7e6e7fc33b3592ec7R355-R357